### ISSUE TITLE
Decide on folder structure of DAM backend

### ISSUE THREAD

**Issue by georgemillard** [emfoundation/asset-manager#289]
When a Content Owner creates an Asset on the DAM, they will be organised in to Folders, in a similar way to files on a hard drive.

The folders should be MICE (mutually exclusive and collectively exhaustive), so everything has a place, and nothing is duplicated.

What is the most useful folder structure to implement?

For example, should the top level be Format, Content Type or Project (not Project, I think as some Assets would sit in several projects...)

@MollyM42 @dalwal08 @asimonw @fannyBkane (+ Ian when he gets a GitHub account) discuss :smile:

**Comment by dalwal08** [comment]
I think format would be best. Organising folders by Type could really confuse things as there may be a temptation to classify the Type as "Infographic" but save it in the "Case Studies" folder.

also, format is easy to make MECE whereas Content Type is not, as per the discussion going on in another issue (which I don't know how to link to).

**Comment by ianjamesbanks** [comment]
I agree that format works on the MECE front, but wouldn't you then get folders that wouldn't add much to a list of assets filtered by file type (except that e.g. docs and pdfs would fall under documents say). If a piece of content can only be assigned one type (meaning it will also be MECE) then could we have folders according to that (with the choice of folder being automatically determined by the type - or at least there being no choice to deviate from it)? Or is that not very practical?

**Comment by asimonw** [comment]
MECE is definitely important, but I think quite important as well is what will be most convenient for content managers in their day to day work. Content type is indeed less straight forward to make MECE (or at least define the different groups), but the aim is still to keep it MECE as well. So what's the most useful way to organise assets, by format or content type? My feeling is that it's less useful to have a quick overview of all files of the same format than all files of the same content type, but I'm not going to be managing content (and it will also depend a lot on the nature of the overall content). What do you guys think?

@ianjamesbanks Currently, the format or content type (depending on what we decide on) won't be set automatically based on the top folder. That does add some unnecessary duplication. @georgemillard something to look at doing on save?

@dalwal08 You can link to another issue by typing `#` and then the issue number. A list of suggestions will pop up as well, based on your recent activity.

**Comment by georgemillard** [comment]
@asimonw I plan to implement automatic setting of format based on filetype on save, but doing so for content type may not be possible, as file type won't map directly to content type.

**Comment by georgemillard** [comment]
@asimonw We would have to implement a solution that recognised a format type from its link, but I think that would be fine.

I agree with @asimonw and @ianjamesbanks that Content-Type categorisation provides a good user experience for content owners, many of whom may be responsible for assets of a particular Content-Type, eg case studies...
